subjects/wordflip/README.md

```diff
@@ -0,0 +1,47 @@
+## wordflip
+
+### Instructions
+
+Write a function `WordFlip()` that receives a `string` and returns it in reverse order.
+
+- Prints the Output followed by newline `\n`.
+- If the string is empty, return `Invalid Output`.
+- Ignore spaces if there are more than one space in between words and ignore all spaces at the edge of the `string`.
+
+### Expected function
+
+```go
+func WordFlip(str string) string {
```

Comment:
From this signature it appears that the function should return, but before it's asked to print.